django_db_schema/hospital/models.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Comment:
Define related names (read about it if you aren't familiar with the concept)

Reply:
d